fedbiomed/common/secagg/_additive_ss.py

```diff
@@ -0,0 +1,243 @@
+import random
+from math import log2
+from typing import List, Optional, Union
+
+from fedbiomed.common.exceptions import FedbiomedTypeError, FedbiomedValueError
+
+
+class AdditiveSecret:
+
+    def __init__(self, secret: Union[int, List[int]]) -> None:
+        """
+        Initializes the AdditiveSecret class with the provided additive secret value.
+
+        Args:
+            secret: The secret to be shared, either an integer or a list of integers.
+
+        Raises:
+            FedbiomedValueError: If the secret is not an int or a list of integers.
+        """
+        if not (
+            isinstance(secret, int)
+            or (isinstance(secret, list) and all(isinstance(i, int) for i in secret))
+        ):
+            raise FedbiomedValueError("AdditiveSecret must be an int or a list of int")
+        self.secret = secret
```

Comment:
```suggestion
        self._secret = secret
```

Shouldn't it be a private variable ?